test/index.spec.js

```diff
@@ -62,6 +63,8 @@ function runTestWithConfig (testConfig, initialStoreState, label) {
             return state + 1
           case 'DECREMENT':
             return state - 1
+          case 'DUMMY':
+            return state
```

Comment:
Added action without state changes, oh maybe that was not necessary... Well it nice and verbose now.. right :D ?